src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	private String block;
+	
+	public Block(String block){
+		this.block = block;
+	}
+	
+	public String getBlock(){
+		return block;
+	}
+	
+
+}
```

Comment:
The class is a little bare bone :-). Consider adding validation, toString() methods.